### DISCUSSION TITLE
How much did each run cost you?

### DISCUSSION THREAD

**Discussion by tawsifkamal** [microsoft/graphrag#440] [Q&A]
Was wondering if you all could share the number of tokens and the cost of the Graph creation for the documents you have tested out with! Additionally, the model you used. 

Thanks!

**Comment by zanderjiang** [comment]
I've been using GPT-4-Turbo, it's really really expensive. I've been trying to index 1MB worth of txt files. The result (with a custom prompt) is pretty good, but it costs a lot to create the knowledge graph, and every asearch call is like 0.4 dollars.

**Comment by mystvearn** [comment]
Tried it on 1000-page-PDF extracted content. I forgot to change the model from GPT4-turbo-preview to GPT-4o so it cost me $120 for the whole process

**Reply by mengdahuang** [reply]
Does Graphrag already support PDF files?

**Reply by zanderjiang** [reply]
I think he extracted the content, so maybe stored in a csv or txt.

**Comment by zebathon** [comment]
The demo is $5 out of the box on gpt-4-turbo

**Comment by DanielSchuhmacher** [comment]
related to https://github.com/microsoft/graphrag/discussions/651

**Comment by shaoqing404** [comment]
I trained with deepseek on a dataset totaling about 1 million words, 1.2 million tokens, which cost about $8.20
